scripts/content/generate-docs-content.js

```diff
@@ -0,0 +1,50 @@
+const glob = require("glob");
+const fs = require("fs");
+const parser = require("node-html-parser");
+const TurndownService = require("turndown");
+
+const service = new TurndownService();
+
+async function walkTheDocs() {
+    console.log("Generating docs JSON...")
+
+    const paths = glob.sync("./public/{docs,templates,learn}/**/*.html", { ignore:[ "./public/docs/reference/pkg/**/*" ] });
+    const jsonOutput = [];
+
+    paths.forEach(async path => {
+        const fileContent = fs.readFileSync(path, "utf-8").toString();
+
+        const doc = parser.parse(fileContent);
+        const main = doc.querySelector(".docs-main-content .docs-content");
+
+        // Skip redirects.
+        if (fileContent.includes('meta http-equiv=refresh content="0; url=')) {
+            return;
+        }
+
+        if (main && main.textContent) {
+            jsonOutput.push({
+                path: path.replace("./public", ""),
+                title: doc.querySelector("title").textContent,
+                h1: doc.querySelector("h1").textContent,
+                main: {
+                    raw: main.rawText,
+                    textContent: main.textContent,
+                    markdown: service.turndown(main.innerHTML),
```

Comment:
It looks like markdown links are `[](url)title` instead of `[title](url)`

Reply:
Yeah, I noticed this too. Wonder if it's the Hugo minification that's confusing the parser. 🤔

Reply:
Looks like that was it, yeah; should be better now. (May need to fiddle a bit more to improve rendered Markdown generally, though.)